src/Itinerary.js

```diff
@@ -0,0 +1,12 @@
+const { blue } = require("color-name");
+
+function Itinerary(ports) {
+  this.ports = ports;
+};
+
+
+
+module.exports = Itinerary;
+
+
+
```

Comment:
Theres a lot of empty space here that doesnt need to be. Theres no hard and fast rules with this stuff, but as a convention you should only put a blank line between each block of logic, so after the imports, between each function, and before the `module.exports` statement at the end. You never need more than 1 consecutive blank line in you rcode:
```
function Itinerary(ports) {
  this.ports = ports;
};

module.exports = Itinerary;

```